src/app/components/modules/simplified-lunch/simplified-lunch.component.ts

```diff
@@ -0,0 +1,124 @@
+import { Component, Input, OnInit, OnDestroy, ElementRef, ViewChild } from '@angular/core';
+import moment from 'moment';
+import * as ElementQueries from 'css-element-queries/src/ElementQueries';
+
+import { AlertService } from '../../../services/alert.service';
+import { LunchService } from '../../../services/lunch.service';
+import { UserService } from '../../../services/user.service';
+
+
+@Component({
+  selector: 'mymicds-simplified-lunch',
+  templateUrl: './simplified-lunch.component.html',
+  styleUrls: ['./simplified-lunch.component.scss']
+})
+export class SimplifiedLunchComponent implements OnInit {
+
+  	loading = true;
+  	lunchDate = moment();
+  	lunch = [];
+    todaysLunch = null;
+  	schools = [
+  		'upperschool',
+  		'middleschool',
+  		'lowerschool'
+  	];
+  	school = this.schools[0];
+    @ViewChild('moduleContainer') containerEl: ElementRef;
```

Comment:
It doesn't look like this `containerEl` is being used anywhere else in the class or in the HTML. This may not be necessary and is usually used with the `css-element-queries`'s `ResizeSensor`, which isn't used here.